Regression/Checksum/benchmarks_json/Python_PlasmaAccelerationMR.json

```diff
@@ -15,28 +15,28 @@
   },
   "lev=0": {
     "Ex": 6818198913.113014,
-    "Ey": 6818198913.112965,
```

Comment:
Do you understand why these values are changing by so much?

Reply:
yeah.. my pywarpx build was not uptodate. I am fixing it now.